webapp/src/components/App/Singlestudy/explore/Configuration/General/dialogs/ScenarioBuilderDialog/tabs/Thermal.tsx

```diff
@@ -0,0 +1,66 @@
+import { useContext, useEffect, useState } from "react";
+import useStudySynthesis from "../../../../../../../../../redux/hooks/useStudySynthesis";
+import { getAreas } from "../../../../../../../../../redux/selectors";
+import PropertiesView from "../../../../../../../../common/PropertiesView";
+import SplitLayoutView from "../../../../../../../../common/SplitLayoutView";
+import UsePromiseCond from "../../../../../../../../common/utils/UsePromiseCond";
+import ListElement from "../../../../../common/ListElement";
+import ConfigContext from "../ConfigContext";
+import Table from "./Table";
+
+interface Props {
+  nbYears: number;
+}
+
+function Thermal(props: Props) {
+  const { nbYears } = props;
+  const { studyId } = useContext(ConfigContext);
+  const res = useStudySynthesis({ studyId, selector: getAreas });
+  const [selectedAreaId, setSelectedAreaId] = useState("");
+
+  useEffect(
+    () => {
+      const { data } = res;
+      setSelectedAreaId(data && data.length > 0 ? data[0].id : "");
```

Comment:
Optional: `setSelectedAreaId((data?.length && data[0]?.id) || "");` you can use optional chaining